server/service/src/sync/central_data_synchroniser.rs

```diff
@@ -89,7 +98,6 @@ fn insert_one_and_update_cursor(
     row: &SyncBufferRow,
     cursor: u32,
 ) -> Result<(), RepositoryError> {
-    // note: if already in a transaction this creates a safepoint:
```

Comment:
i don't think this comment was accurate, as it was using transaction_sync not _etc